src/app.js

```diff
@@ -1 +1,40 @@
 'use strict';
+
+const generateRandomNumber = require('./numberGenerator');
+const calculateBullsAndCows = require('./bullsAndCowsCalculator');
+const { askQuestion, close } = require('./io');
+
+async function playGame() {
+  const secretNumber = generateRandomNumber();
+  let tries = 0;
+
+  /* eslint-disable no-console */
+  console.log('Welcome to Bulls and Cows! Try to guess the 4-digit number.');
+
+  while (true) {
```

Comment:
while(true) is a infinity loop, so it's a bad practice 
so, make sense to add condition for this cycle